src/lib/apimanifest.csproj

```diff
@@ -30,6 +31,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.OpenApi.Readers" Version="1.6.11" />
+    <PackageReference Include="System.Text.Json" Version="8.0.0" />
```

Comment:
this should be a bracket dependency so it works on lower runtimes

Reply:
5 might bring a broader compatibility with non Microsoft runtimes. Even though the use case might be very narrow.

Reply:
v5.x of the lib has been deprecated - https://www.nuget.org/packages/System.Text.Json/5.0.0. v6 seems appropriate for now.